Gems/Atom/Feature/Common/Editor/Scripts/ColorGrading/from_3dl_to_azasset.py

```diff
@@ -0,0 +1,114 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Takes a 3dl file and encodes it into a .azasset file that can be read by the
+# AssetProcessor
+
+import json
+import sys, getopt
+import logging as _logging
+
+# ------------------------------------------------------------------------
+_MODULENAME = 'ColorGrading.from_3dl_to_azasset'
+
+import ColorGrading.initialize
+ColorGrading.initialize.start()
+
+_LOGGER = _logging.getLogger(_MODULENAME)
+_LOGGER.debug('Initializing: {0}.'.format({_MODULENAME}))
+
+try:
+    import OpenImageIO as oiio
+    pass
+except ImportError as e:
+    _LOGGER.error(f"invalid import: {e}")
+    sys.exit(1)
+# ------------------------------------------------------------------------
+
+DEFAULT_AZASSET_HEADER = """\
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "LookupTableAsset",
+    "ClassData": {
+        "Name": "LookupTable",
+        "Intervals": [
+}
```

Comment:
Based on how your appending intervals after the header, I think this } would need to be removed or all your interval numbers would come after the } and the whole json would be malformed. Another option would be to write the whole json string with entries for the intervals and values:
```
AZASSET_LUT = """\
{
    "Type": "JsonSerialization",
    "Version": 1,
    "ClassName": "LookupTableAsset",
    "ClassData": {
        "Name": "LookupTable",
        "Intervals": %s,
        "Values": %s
    }
}
```
Then you create your strings of values and intervals, and when you want to composite the whole file, just 
```
AZASSET_LUT % (intervals, values)
```
It'd trim this function down to like 4 or 5 lines.

Reply:
That makes more sense, I'll also make it more py3 ready and use an f'string'

Reply:
`def write_azasset(file_path, lut_size, lut_intervals, lut_values, azasset_json=AZASSET_LUT):

    values_str = ''
    for idx, px in enumerate(lut_values):
        values_str += "{0}, {1}, {2}\n".format(px[0], px[1], px[2])

    intervals_str = ', '.join(map(str, lut_intervals))
    
    azasset_json = azasset_json % (intervals_str, values_str)
    
    asset_file_path = f"{file_path}.azasset"
    _LOGGER.info(f"Writting {asset_file_path}...")
    asset_file = open(asset_file_path, 'w')
    asset_file.write(azasset_json)`